security.md

```diff
@@ -67,6 +68,16 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2016-9593"></a>CVE-2016-9593: foreman-debug collects sensitive data
+
+It was found that foreman-debug did not obfuscate sensitive information (such as passwords) from the katello-installer log file, allowing a user authorized to access the log files created by foreman-debug to gain access to potentially sensitive information.
```

Comment:
1. foreman-installer
1. Also answer and other configuration files, not only log files.
1. "the log files created by foreman-debug" should be "archives created by foreman-debug".